src/python/nxt_python.c

```diff
@@ -373,6 +375,33 @@ nxt_python_set_target(nxt_task_t *task, nxt_python_target_t *target,
         goto fail;
     }
 
+    value = nxt_conf_get_object_member(conf, &prefix_str, NULL);
```

Comment:
Maybe this whole code block could go into a separate helper function?  This function is already growing more than I'd like, I think.

Reply:
Is it really a "helper" function if it is only called once from a single place?

Reply:
It's not a bad thing, IMHO.  If we know it's only called at one place and inconditionally, we can make it `[[gnu::always_inline]]` to help the compiler (in case we can prove it doesn't inline it on its own).

We also get the opportunity to name the whole section of code, which might increase readability.  Unless you consider that moving it into a separate function would hurt readability because of loosing context.

Reply:
It would also simplify error handling.

Reply:
I'm not entirely happy about this:

```c
                nxt_free(prefix);
                goto fail;
```

Reply:
I moved the code into a separate function. All the tests pass, but please let me know if you notice any problems with my changes.

Reply:
Thanks for the changes, and thanks for keeping them as a separate patch (it made the differential review much easier).  They look good (still can improve a little bit, but great job!).